packages/client/ui/react-native-ui/src/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from "./hooks/index";
+export * from "./providers/index";
+
+export { CrossmintEvents, useCrossmintEvents } from "@crossmint/client-sdk-base";
```

Comment:
why do we need this?

Reply:
removed https://github.com/Crossmint/crossmint-sdk/pull/1020/commits/f9959cd39c36bab6079d6b78ed1aeeb5962787cf